redundant-bmc/src/redundancy_mgr.cpp

```diff
@@ -187,7 +199,7 @@ void RedundancyMgr::systemStateChange(SystemState newState)
 
     systemState = newState;
 
-    // TODO: on any change recalculate failovers paused
+    determineAndSetFailoversPaused();
```

Comment:
https://github.com/ibm-openbmc/phosphor-state-manager/pull/84/commits/2cd9b6fb1f06f63a917a8f1e4b36decfa0450683#diff-bc7c6d2f949016fa3396798fab8c9c3c2ed18bbc298a323d6a01b4717ae11469L169 is it needed, now?

Reply:
I can't quite follow what you mean here...  Can you elaborate?

Reply:
Oh, it looks like the link to the untouched code isn't displaying correctly.

What I meant to say is:
In this function, I noticed the following snippet:

```
else if (newState == SystemState::booting)
{
    // TODO: failovers paused, etc
}

```

I think this block may be unnecessary now, since you're recalculating the failover points (FOP) anyway.

Reply:
Thanks, yea I will remove that.

Reply:
Fixed.